tests/keras_contrib/backend/test_backends.py

```diff
@@ -99,6 +100,32 @@ def test_extract(self):
                 assert zth.shape == ztf.shape
                 assert_allclose(zth, ztf, atol=1e-02)
 
+    def test_moments(self):
+        input_shape = (10, 10, 10, 10)
+        x_0 = np.zeros(input_shape)
+        x_1 = np.ones(input_shape)
+        x_random = np.random.random(input_shape)
+
+        th_axes = [0, 2, 3]
+        tf_axes = [0, 1, 2]
```

Comment:
Ah no the tests were failing cause of extremely small tolerance. They are passing now.